kbfsgit/runner.go

```diff
@@ -734,7 +800,7 @@ func (r *runner) handleClone(ctx context.Context) (err error) {
 		r.errput.Write([]byte("done." + elapsedStr + "\n"))
 
 		sw = &statusWriter{r, nil, 0, b, 0}
-		r.errput.Write([]byte("Cloning: "))
+		r.errput.Write([]byte("Cryptographic cloning: "))
```

Comment:
Assuming "cryptographic" is adjective, shouldn't it be "cryptographically" here?

Reply:
This was suggested by @malgorthms, but I think it's fine.  It's just adding "-ing" to "cryptopgraphic clone", so "cryptographic" is still just an adjective, not an adverb.